src/dx/publish/mod.rs

```diff
@@ -91,6 +93,30 @@ impl<T> PubNubClient<T> {
     }
 }
 
+impl<T, M, D> PublishMessageViaChannelBuilder<T, M, D>
+where
+    M: Serialize,
+    D: for<'de> Deserializer<'de, PublishResponseBody>,
+{
+    fn prepare_context_with_request(
+        self,
+    ) -> Result<PublishMessageContext<T, D, TransportRequest>, PubNubError> {
+        let instance = self
+            .build()
+            .map_err(|err| PubNubError::PublishError(err.to_string()))?;
```

Comment:
this is gonna change right? using new convention

Reply:
You mean the PAMv3? Or have I missed something?